platform/broadcom/saibcm-modules/debian/opennsl-modules-4.9.0-7-amd64.init

```diff
@@ -54,7 +54,9 @@ start)
 stop)
```

Comment:
Would it make sense to change opennsl-modules-4.9.0-7-amd64.service so there is no ExecStop action for the service?  This script can still be used if we wanted to remove kernel modules for some cases.

Reply:
Thanks for the great idea! I've made the change.